umbral-pre-python/umbral_pre/__init__.pyi

```diff
@@ -201,35 +208,34 @@ class VerifiedCapsuleFrag:
     def to_bytes_simple(self) -> bytes:
         ...
 
-
-def reencrypt(capsule: Capsule, kfrag: VerifiedKeyFrag) -> VerifiedCapsuleFrag:
-    ...
-
-
 def decrypt_reencrypted(
         receiving_sk: SecretKey,
         delegating_pk: PublicKey,
         capsule: Capsule,
-        cfrags: Sequence[VerifiedCapsuleFrag],
+        verified_cfrags: Sequence[VerifiedCapsuleFrag],
         ciphertext: bytes,
-        ) -> Optional[bytes]:
+) -> Optional[bytes]:
     ...
 
+def reencrypt(capsule: Capsule, kfrag: VerifiedKeyFrag) -> VerifiedCapsuleFrag:
+    ...
 
-class CurvePoint:
 
-    def coordinates(self) -> Tuple[bytes, bytes]:
-        ...
+@final
+class CurvePoint:
+    coordinates: Tuple[bytes, bytes]
```

Comment:
This is actually `Optional[Tuple[bytes, bytes]]`.

Reply:
:heavy_check_mark:

Reply:
Is this the only member of that struct? If so, why is it Optional?

Reply:
It's `None` if the point is an infinity point.